shared/messaging/proto/pocket_envelope.proto

```diff
@@ -8,4 +8,5 @@ option go_package = "github.com/pokt-network/pocket/shared/messaging";
 
 message PocketEnvelope {
   google.protobuf.Any content = 1;
+  uint64 nonce = 2; // TECHDEBT: should this be the same as the nonce in `Transaction`?
```

Comment:
:+1: I'm not super familiar with the background here, this is a comment that has been transported and transformed across history. AFAIK discussion #278 has the most detail on this:

```suggestion
  uint64 nonce = 2; // DISCUSS(#278): should this be the same as the nonce in `Transaction`?
```